src/WordSelector/WordSelector.js

```diff
@@ -38,7 +38,7 @@ export class WordSelector extends Component {
               placeholder="Submit"
               onClick={(e) => {
                 this.makeWordToGuess(e)
-                }}
+              }}
```

Comment:
really cutting down on data loss here. We have to do these things or big data wins